kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/ModemManagerDbusWrapper.java

```diff
@@ -272,4 +284,84 @@ protected void resetHandlersDisable(String deviceId) {
             }
         }
     }
+
+    protected void failedResetHandlerEnable(String deviceId, Optional<String> modemManagerDbusPath, int delayMinutes)
```

Comment:
I would rethink this method name. This is not a `ResetHandlerEnable`... this is just scheduling the reset.